lib/twitter-ads/resources/dsl.rb

```diff
@@ -20,6 +20,7 @@ module InstanceMethods
       #
       # @since 0.1.0
       def from_response(object)
+        object = object[0] if object.is_a? Array
```

Comment:
Be careful about this change, some API objects do deserialize to an array and this will cause problems. I wouldn't advise making this change unless you're using a modified version of the SDK for limited purposes and this suits that purpose. If you made this change to address the issue with `tweet_ids` in `PromotedTweet` you can revert this change and just update to v0.2.4.